lib.nix

```diff
@@ -25,6 +25,7 @@ let
             # Filter out cabal build products.
             baseName == "dist" || baseName == "dist-newstyle" ||
             baseName == "cabal.project.local" ||
+            lib.hasPrefix ".ghc.environment" baseName ||
```

Comment:
This is one of the ideocities, I don't get... this should *never* have been added to ghc without a flag. Ahh well here we are.